vcluster/_fragments/integrations/cert-manager.mdx

```diff
@@ -0,0 +1,197 @@
+import Flow, { Step } from '@site/src/components/Flow'
+import NavStep from '@site/src/components/NavStep'
+import Button from '@site/src/components/Button'
+import Label from '@site/src/components/Label'
+import Field from '@site/src/components/Field'
+
+import CertManagerPartial from '../../_partials/config/integrations/certManager.mdx';
+
+import BasePrerequisites from '../../../platform/_partials/install/base-prerequisites.mdx';
+
+import CodeBlock from '@theme/CodeBlock';
+
+import Deploy from '../../_partials/deploy/deploy.mdx'
+
+import ProAdmonition from '../../_partials/admonitions/pro-admonition.mdx'
+
+<ProAdmonition/>
+
+### Prerequisites
+
+<BasePrerequisites />
+
+- `cert-manager` operator installed on your host cluster. See instructions at https://cert-manager.io
+
+## Cert manager integration
+
+To enable the cert-manager integration, set the following fields as shown below:
+
+
+```yaml title="Enable cert-manager integration"
+integrations:
+  certManager:
+    enabled: true
+```
+
+This configuration enables the integration, imports cluster-scoped ClusterIssuers from the host cluster into the virtual cluster, and exports namespaced Issuers and Certificates from the virtual cluster into the host cluster.
+
+<Flow id="cert-manager-integration">
+ <Step>
+
+  Create the Issuer
```

Comment:
This seems to be a ClusterIssuer that's being created on the host cluster.

1. Can we describe that it's a ClusterIsssuer (the actual CR that's being created)?
2. Can we describe when we apply it, it'll be in the host cluster?